publisher-sdk/src/main/java/com/criteo/publisher/advancednative/CriteoNativeLoader.java

```diff
@@ -113,16 +114,16 @@ public View createEmptyNativeView(@NonNull Context context, @Nullable ViewGroup
    */
   public void loadAd() {
     try {
-      doLoad();
+      doLoad(new ContextData());
```

Comment:
nit: some static factory method such as `none()` would be nice from a readability perspective: `doLoad(ContextData.none())`

Reply:
From a `ContextData.none()` I would expect it to be a constant. But this object is mutable so it would be possible to do `ContextData.none().set("toto", "tata")`. Also, this case is kind of special and I don't expect publishers to use it since there is still the API without context.
For test, I plan to elaborate the given context objects so there would be less `new ContextData()` alone.